python/connectors/connectors_auto_start.py

```diff
@@ -0,0 +1,121 @@
+# Checks if connector has run in the last 24 hours.  If not, launches it.
+
+import os
+import sys
+import json
+import requests
+from datetime import datetime, timedelta
+from prettytable import PrettyTable
+
+# Returns connector information from the List Connectors API.
+def get_connectors(base_url, headers):
+    connectors = []
+    list_connectors_url = base_url + "connectors"
```

Comment:
This seems to mix and match string concatenation and interpolation. I would say we should switch this line to use interpolation the same way it is used on line 17. I.E. f"{base_url}connectors".

Reply:
I fixed all the string concatenations I could find.